README.md

```diff
@@ -9,10 +8,18 @@ You need to have `docker` and `docker-compose`installed
 
 You can run this project by typing `make` on your Terminal
 
-
 ### Troubleshooting
 
 Currently this example project uses local repos as dependencies, exposing them from your computer to
 the docker container. If the SDK is not found, you can check `docker-compose.yml` on the repo's root and check the
 `volumes` entry, to verify that the origin path (the one with `{$HOME}`) matches the location of the SDK on your local
-machine
\ No newline at end of file
+machine
+
+### Alternative Way to run the example project
+
+If you don't want to use docker, you can run the example project by following these steps:
+
+- Execute the command python3 -m venv venv
+- Activate the virtual environment with source venv/bin/activate
+- Install the dependencies with pip install transbank-sdk
```

Comment:
this installs only the transbank-sdk, but you need all from requirements.txt
replace this command with _pip install -r requirements.txt_